content/en/docs/concepts/storage/volumes.md

```diff
@@ -26,85 +26,81 @@ Familiarity with [Pods](/docs/concepts/workloads/pods/pod/) is suggested.
-somewhat looser and less managed.  In Docker, a volume is simply a directory on
-disk or in another Container.  Lifetimes are not managed and until very
-recently there were only local-disk-backed volumes.  Docker now provides volume
+somewhat looser and less managed. In Docker, a volume is simply a directory on
+disk or in another container. Lifetimes are not managed and until very
+recently there were only local-disk-backed volumes. Docker now provides volume
 drivers, but the functionality is very limited for now (e.g. as of Docker 1.7
-only one volume driver is allowed per Container and there is no way to pass
+only one volume driver is allowed per container and there is no way to pass
 parameters to volumes).
 
 A Kubernetes volume, on the other hand, has an explicit lifetime - the same as
-the Pod that encloses it.  Consequently, a volume outlives any Containers that run
-within the Pod, and data is preserved across Container restarts. Of course, when a
-Pod ceases to exist, the volume will cease to exist, too.  Perhaps more
-importantly than this, Kubernetes supports many types of volumes, and a Pod can
+the pod that encloses it. Consequently, a volume outlives any containers that run
+within the pod, and data is preserved across container restarts. Of course, when a
+pod ceases to exist, the volume is destroyed. Perhaps more
+importantly than this, Kubernetes supports many types of volumes, and a pod can
 use any number of them simultaneously.
 
 At its core, a volume is just a directory, possibly with some data in it, which
-is accessible to the Containers in a Pod.  How that directory comes to be, the
+is accessible to the containers in a pod.  How that directory comes to be, the
 medium that backs it, and the contents of it are determined by the particular
 volume type used.
 
-To use a volume, a Pod specifies what volumes to provide for the Pod (the
-`.spec.volumes`
-field) and where to mount those into Containers (the
-`.spec.containers[*].volumeMounts`
-field).
-
+To use a volume, a `Pod` specifies what volumes to provide for the `Pod`
+by setting the `.spec.volumes` field and where to mount those volumes
+into containers (the `.spec.containers[*].volumeMounts` field).
 A process in a container sees a filesystem view composed from their Docker
-image and volumes.  The [Docker
-image](https://docs.docker.com/userguide/dockerimages/) is at the root of the
-filesystem hierarchy, and any volumes are mounted at the specified paths within
-the image.  Volumes can not mount onto other volumes or have hard links to
-other volumes.  Each Container in the Pod must independently specify where to
+image and volumes. The [Docker image](https://docs.docker.com/userguide/dockerimages/)
+is at the root of the filesystem hierarchy. Volumes mount at the specified paths within
+the image. Volumes can not mount onto other volumes or have hard links to
+other volumes. Each container in the pod must independently specify where to
```

Comment:
Do you think this is referring to the Kubernetes Container resource or containers in general?

Reply:
I agree that this is sentence is confusing. I've gone over this a few times.
I will update and capitalize as a resource because the sentence is describing how to "specify" the Container and Pod objects.
I also agree that the name of the field is called `container` but if this styling is used I would update the sentence to identify the "field to set".